crudOperations/storeOperations/storeRouter.js

```diff
@@ -22,7 +22,7 @@ router.post("/", async (req, res) => {
           console.log(user)
           const storeWithEmail = {
             store_name: store.store_name,
-            userID: user.id,
```

Comment:
May lead to store not being created

Reply:
Works locally when I create a store on local database but not live database.  Live database still has the user.id